src/slic3r/GUI/PrintHostDialogs.hpp

```diff
@@ -131,6 +131,45 @@ class PrintHostQueueDialog : public DPIDialog
     bool load_user_data(int, std::vector<int>&);
 };
 
+class ElegooPrintHostSendDialog : public GUI::MsgDialog
```

Comment:
Ref to my another comment, can this class inherite from printHotSendDialog instead?